openqa-advanced-retrigger-jobs

```diff
@@ -8,23 +8,62 @@ additional_filters="${additional_filters+" and $additional_filters"}"
 comment="${comment:-""}"
 max_jobs_per_request=25
 dry_run="${dry_run:-"0"}"
-[ "$dry_run" = "1" ] && client_prefix="echo"
-sql_command="select id from jobs where (${worker_string}${result} and clone_id is null and t_finished >= '$failed_since'$additional_filters);"
-# shellcheck disable=SC2029
-job_ids=${JOB_IDS:-$(ssh "$host" "sudo -u geekotest psql --no-align --tuples-only --command=\"$sql_command\" openqa")}
-
-query_params=()
-[[ $cli_protocol ]] && host=$cli_protocol://$host
-[[ $cli_port ]] && host+=:$cli_port
+
+usage() {
+    cat << EOF
+Usage: $0 [OPTIONS]
+
+Retrigger openQA jobs based on database queries.
+
+By default retriggers openQA jobs with result '$result' since '$failed_since'
```

Comment:
by default is the today day but this will print the date right? Should we be more explicit here? not sure

Reply:
Sorry, those are unrelated changes that shouldn't be in this PR. Removed